cfgov/unprocessed/apps/teachers-digital-platform/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "teachers-digital-platform",
+  "description": "Youth financial education activity search and building blocks journey tool",
+  "homepage": "http://localhost:8000/practitioner-resources/youth-financial-education/",
+  "author": {
+    "name": "Consumer Financial Protection Bureau",
+    "email": "tech@cfpb.gov",
+    "url": "https://cfpb.github.io/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cfpb/cfgov-refresh.git"
+  },
+  "license": "SEE LICENSE IN TERMS.md",
+  "engines": {
+    "node": ">=10.x"
+  },
+  "devDependencies": {
+    "aos": "3.0.0-beta.6",
+    "cf-atomic-component": "8.0.1",
+    "cf-buttons": "8.0.1",
+    "cf-core": "8.0.1",
+    "cf-expandables": "8.0.1",
+    "cf-forms": "8.0.2",
+    "cf-grid": "8.0.1",
+    "cf-icons": "8.0.1",
+    "cf-layout": "8.0.1",
+    "cf-notifications": "8.0.1",
+    "cf-pagination": "8.0.1",
+    "cf-tables": "8.0.1",
+    "cf-typography": "8.0.1",
```

Comment:
Are the list of `cf-` packages here unused?

Reply:
Whoops, yes, thanks. Will remove.